build.gradle

```diff
@@ -595,10 +595,10 @@ configure(rootProject) {
 
 	gitPublishCommit.dependsOn(prepareDocsForUploadToGhPages)
 	gitPublishCommit.dependsOn(createCurrentDocsFolder)
+}
 
-	task wrapper(type: Wrapper) {
-		description = 'Generates gradlew and gradlew.bat scripts'
-		gradleVersion = '4.0.2'
-	}
-
+task wrapper(type: Wrapper) {
+	description = 'Generates gradlew and gradlew.bat scripts'
+	gradleVersion = '4.1'
+	distributionType = Wrapper.DistributionType.ALL
```

Comment:
It was changed from "all" to "bin" some days ago via https://github.com/junit-team/junit5/commit/719127f615fce25a32fc4ed90514d8a748add9c5#diff-fd9d61e37d939c941e7a62077f38d62d -- please revert it as well.